front/app/modules/commercial/analytics/admin/components/ActiveUsersCard/renderTooltip.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+import { Tooltip } from 'recharts';
+import TooltipOutline from 'components/admin/Graphs/utilities/TooltipOutline';
+
+// styling
+import { colors } from 'components/admin/Graphs/styling';
+
+// i18n
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+// utils
+import { toFullMonth } from 'utils/dateUtils';
+
+// typings
+import { IResolution } from 'components/admin/ResolutionControl';
+
+type CustomTooltipProps = {
+  label?: string;
+  payload?:
+    | [
+        {
+          payload?: {
+            activeUsers: number;
+            date: string;
+          };
+        }
+      ]
+    | [];
+  resolution: IResolution;
+};
+
+const CustomTooltip = ({ label, payload, resolution }: CustomTooltipProps) => {
+  if (!payload?.[0]?.payload || !label) return null;
+
+  return (
+    <TooltipOutline label={toFullMonth(label, resolution)}>
+      <Box py="0px">
+        <FormattedMessage {...messages.activeUsers} />:{' '}
+        {payload[0].payload.activeUsers}
+      </Box>
+    </TooltipOutline>
+  );
+};
+
+const renderTooltip = (resolution: IResolution) => (props) =>
+  (
+    <Tooltip
+      {...props}
+      cursor={{ stroke: colors.gridHoverColor }}
+      content={(props) => (
+        <CustomTooltip
+          label={props.label}
+          payload={props.payload as any}
```

Comment:
omg, the forbidden move, you will be cursed by the type gods to never know the difference between almond milk and cow milk.

Reply:
Happy to hear how to fix this, since `recharts` is not exporting the types I need here

Reply:
just kidding, but thinking about it, isn't this suppose to be `TimeSeries` (from `hooks/useActiveUsers/typings`) ??

Reply:
Unfortunately not, it's some weird object generated by `recharts` that differs per chart